### ISSUE TITLE
Adding more transformation to make model training harder to overfit on the training data.

### ISSUE THREAD

**Issue by Shrey-Viradiya** [Shrey-Viradiya/Pneumonia_Detection#18]
@tirthasheshpatel can you please see and add some more transformation other than provided in the model.py to make the model training process harder. It will be helpful for us to avoid overfitting. I already added some and got some good results. Remember initially we were obtaining 98 % trining accuracy in the 1st epoch. Please add more..
[https://pytorch.org/docs/stable/torchvision/transforms.html](https://pytorch.org/docs/stable/torchvision/transforms.html)
